bundles/framework/admin-users/view/RoleBlock.jsx

```diff
@@ -1,14 +1,23 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { TextInput, Message } from 'oskari-ui';
 import { UserOutlined } from '@ant-design/icons';
 import { Block, Button, ButtonContainer } from './styled';
 
+const ADMIN = 'admin';
+
 export const RoleBlock = ({ role, controller, isSystemRole, editingRole }) => {
-    const { id, name } = role;
+    const { id, name, type } = role;
     if (isSystemRole) {
         return (
             <Block>
                 <span>{name}</span>
+                { type === ADMIN &&
```

Comment:
Only admin users can be shown like this? This might be better with `where type not in (GUEST, LOGGED_IN)` just to get the long listings removed?